assets/add_new_asset.md

```diff
@@ -1,8 +1,22 @@
 # Trust Wallet Assets Info
+
+```diff
+- On November 8th 2019 we made big changes to the repository, please read summary and recomnadations
+```
+
+### What changed:
```

Comment:
this is not needed here, keep it simple

you can add to release notes on github instead

Reply:
https://github.com/trustwallet/developer/pull/63/commits/53df752447fd2e3a8b755a2dff75b3b90fea70d4